ivadomed/utils.py

```diff
@@ -277,6 +277,58 @@ def get_arguments(parser, args):
     return args
 
 
+def add_version_info(context):
+    path_data = context.get(ConfigKW.LOADER_PARAMETERS).get(LoaderParamsKW.PATH_DATA)
+
+    context[ConfigKW.IVADOMED_VERSION] = _version_string()
+    datasets_version = get_datasets_versions(context)
+    datasets_dict = {}
+
+    path_data = format_path_data(path_data)
+    for i_dataset in range(len(path_data)):
+        origin = get_origin(path_data[i_dataset])
+        if origin != "?!?":
+            datasets_dict[origin] = datasets_version[i_dataset]
+        else:
+            datasets_dict[path_data[i_dataset]] = "data_is_not_annexed"
+
+    context[ConfigKW.DATASETS_VERSION] = datasets_dict
+
+
+def get_datasets_versions(context):
+    path_data = context.get(ConfigKW.LOADER_PARAMETERS).get(LoaderParamsKW.PATH_DATA)
```

Comment:
This is generally not a huge problem in this particular ivadomed context, but be aware that chained `get` could throw None type has no .get method issue.